automation/docker/run-tests.sh

```diff
@@ -4,12 +4,19 @@
 NUM_NODES="${1:-4}"  # default to 4
 ENV="${2:-dev}"  # default to dev
 export ENV=$ENV
+HUB_COMPOSE=hub-compose.yml
 
 #if test runs against a remote FIAB on a GCE node, put IP in param 3
 #if test runs against a local FIAB on a Dokcer, put "local" in param 3
```

Comment:
not your typo but there's a typo in "Docker"

Reply:
nice catch